site/docs/model-audit/usage.md

```
@@ -405,6 +405,34 @@
+   Error: Insufficient disk space for download
+   ```
+
+   Solution: Free up disk space or use a different cache directory:
+
+   ```bash
+   promptfoo scan-model s3://bucket/model.bin --cache-dir /mnt/large-disk/cache
+   ```
+
+7. **License Compliance Failures**
+
+   ```
```

markdownlint-cli2 (0.17.2)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)